aws/templates/resource/resource_ecs.ftl

```diff
@@ -159,7 +169,16 @@
                     }
                 },
                 networkMode == "awsvpc"
-            ) + 
+            ) +
+            valueIfTrue(
+                {
+                    "SchedulingStrategy" : "DAEMON"
+                },
+                (placement.Strategy?has_content) && (placement.Strategy == "daemon"),
```

Comment:
Strategy has a default so should just be able to do straight comparison

Reply:
Fixed